cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/SteadyStateHypothesisExport.java

```diff
@@ -343,10 +351,12 @@ private static void writeStaticVarCompensators(Network network, String cimNamesp
                 // Regulating control could be reactive power or voltage
                 double targetValue;
                 String multiplier;
-                if (regulationMode == StaticVarCompensator.RegulationMode.VOLTAGE) {
+                if (regulationMode == StaticVarCompensator.RegulationMode.VOLTAGE
```

Comment:
I don't understand here... if the static var compensator is `OFF`, the target value can be valid for VOLTAGE and for REACTIVE POWER. For example, '225' can match for both. What we are doing is totally arbitrary...

Reply:
We are assuming if the svc has a valid voltage setpoint (there is a typo in the name of the method) the regulating control is define for regulating voltage. If there  is a valid value for reactive power setpoint, we assume the RC is defined for regulating reactive power.

Reply:
You are right that we are assuming the RC is defined (in the EQ) in a consistent way with the current values of the setpoints. To be reviewed ...

Reply:
I don't understand. When we are `OFF`, it could be a change in IIDM network. We can be voltage or reactive power at import, right? For me the method `isValidSvcVoltageSetpoint` is not enough to say that it was regulating voltage. And it does not rely on import data. Am I wrong?